src/argilla/client/feedback/dataset.py

```diff
@@ -759,13 +759,12 @@ def format_as(self, format: Literal["datasets"]) -> "Dataset":
                     dataset[question.name].append(
                         [
                             {
-                                "user_id": r.user_id,
-                                "value": r.values[question.name].value,
-                                "status": r.status,
+                                "user_id": r.user_id or None,
+                                "value": r.values[question.name].value or None,
```

Comment:
Since some questions can be optional, I think we should prevent this.
```suggestion
                                "value": r.values.get(question.name) or None, # TODO: we should fetch first the response and then the value
```